newsroom/users/users.py

```diff
@@ -19,6 +20,32 @@
 from newsroom.companies.utils import get_company_section_names, get_company_product_ids
 
 
+class UserAuthentication(SessionAuth):
+    def authorized(self, allowed_roles, resource, method):
+        if super().authorized(allowed_roles, resource, method):
```

Comment:
Should this rather be `if not super().authorized(allowed_roles, resource, method): return False`?

Reply:
if user is authorized because has the required role needed I return True, here I'm handling the case when user is not authorized via his role but it might be his profile or a profile from his company if he is an admin.

but I should add check for authentication here, that's missing